packages/sdk/src/BoostCore.ts

```diff
@@ -527,13 +527,22 @@ export class BoostCore extends Deployable<
           Object.keys(this.addresses).map(Number),
         );
       const testnet = chain.testnet || chain.id === 31337;
-      payload.validator = this.SignerValidator({
-        signers: [
-          (testnet
-            ? BoostValidatorEOA.TESTNET
-            : BoostValidatorEOA.MAINNET) as unknown as Address,
-        ],
+      const signers = [
+        (testnet
+          ? BoostValidatorEOA.TESTNET
+          : BoostValidatorEOA.MAINNET) as unknown as Address,
+      ];
+
+      // This seemed like the best approach - I didn't want to use the testnet PK even in local testing
+      // May be another approach but this works for now and is pretty well isolated
+      if (chain.id === 31337) {
+        signers.push('0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266');
+      }
```

Comment:
Yeah I'd prefer this live outside the sdk itself, but if consumers find it useful for prototyping, it's totally defensible.